apps/condo/domains/notification/utils/testSchema/index.js

```diff
@@ -87,9 +93,64 @@ async function resendMessageByTestClient (client, message, extraAttrs = {}) {
     return [data.result, attrs]
 }
 
+async function createTestPushToken (client, extraAttrs = {}) {
+    if (!client) throw new Error('no client')
+    const sender = { dv: 1, fingerprint: faker.random.alphaNumeric(8) }
+    const deviceId = faker.random.uuid()
+    const token = faker.random.uuid()
+    const serviceType = getRandomItem(DEVICE_SERVICE_TYPES_KEYS)
+
+    const attrs = {
+        dv: 1,
+        sender,
+        deviceId, token, serviceType,
+        ...extraAttrs,
+    }
+    const obj = await PushToken.create(client, attrs)
+
+    return [obj, attrs]
+}
+
+async function updateTestPushToken (client, id, extraAttrs = {}) {
+    if (!client) throw new Error('no client')
+    if (!id) throw new Error('no id')
+    const sender = { dv: 1, fingerprint: faker.random.alphaNumeric(8) }
+
+    // TODO(codegen): check the updateTestPushToken logic for generate fields
```

Comment:
Remove this TODO, if you've fulfilled it's requirement.

Reply:
fixed